FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/SchedulerService.cs

```diff
@@ -135,7 +143,7 @@ private async Task<bool> TryAcquireLeaseAsync(CancellationToken cancellationToke
 
                 if (triggerLeaseEntity == null)
                 {
-                    _logger.LogWarning("Try to acquire lease failed, the retrieved lease trigger entity is null.");
+                    _logger.LogInformation("Try to acquire lease failed, the retrieved lease trigger entity is null.");
```

Comment:
this case should not happen, in your design, what is the expected behavior: log error or log information?

Reply:
After offline discussion, we keep log information here since there is no exception here.